broadcast/submit_tx.go

```diff
@@ -15,3 +17,11 @@ type SubmitTxResponse struct {
 	// TxStatus is the status of the transaction.
 	TxStatus TxStatus `json:"txStatus,omitempty"`
 }
+
+// SubmitTxResponse is the response returned by the SubmitBatchTransactions method.
+type SubmitBatchTxResponse struct {
+	// Miner is the URL of the miner that submitted the batch transactions.
+	Miner string `json:"miner"`
+	// SubmitTxResponses are the responses returned by the miner who submitted the batch transactions.
+	SubmitTxResponses []*SubmitTxResponse
```

Comment:
I would change field name to `Data`, `Transactions` or something like that. I don't think using type name as field is a good idea here.